packages/account/src/Sections/Security/Passkeys/components/passkeys-status-layout.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { Fragment } from 'react';
+import classNames from 'classnames';
+import { Button, Text } from '@deriv/components';
+import FormBody from '../../../../Components/form-body';
+import FormFooter from '../../../../Components/form-footer';
+
+export type TPasskeysButtonOnClicks = {
+    onPrimaryButtonClick: () => void;
+    onSecondaryButtonClick?: () => void;
+};
+
+type TPasskeysStatusLayout = {
+    className?: string;
+    description?: React.ReactNode;
+    icon?: React.ReactElement;
+
+    primary_button_disabled?: boolean;
+    primary_button_text: React.ReactElement;
+    scroll_offset?: string;
+    secondary_button_text?: React.ReactElement;
+    title?: React.ReactElement;
+} & TPasskeysButtonOnClicks;
+
+export const PasskeysStatusLayout = ({
+    children,
+    className,
+    description,
+    icon,
+    onPrimaryButtonClick,
+    onSecondaryButtonClick,
+    primary_button_disabled = false,
+    primary_button_text,
+    scroll_offset = '22rem',
```

Comment:
Will `scroll_offset` be the same for mobile and desktop?

Reply:
not sure for now @likhith-deriv , when I work on the card to enable it for desktop I will double check it